ref-schema-url/README.md

```diff
@@ -0,0 +1 @@
+RefSchemaUrl uses a regular expression to validate the urls. If the regex is updated, it can be tested against different valid and invalid strings in `test/regex.spec.js`. It can be tweaked or edited [here](https://regex101.com/r/gM5cU5/1) along with the test cases.
```

Comment:
could you add a link to https://s1seven.github.io/SEP/schemas/#versioning please ?